services/base-jdbc/src/main/resources/org/eclipse/hono/service/base/jdbc/store/device/base.sql.yaml

```diff
@@ -171,3 +171,11 @@ countDevicesOfTenant: |
    SELECT COUNT(*) AS deviceCount FROM %1$s
    WHERE
       tenant_id=:tenant_id
+
+getDevicesOfTenant: |
```

Comment:
Maybe we can already call this *searchDevicesOfTenant* or *findDevicesOfTenant* in anticipation of the search params being supported in the near future?

Reply:
done